### PR TITLE
Using spawn instead of execFile

### PR DESCRIPTION
**Pull request by 2roy999**

### CODE REVIEW

File: dev-scripts/get.js [hunk 1]
```diff
@@ -126,10 +133,10 @@ caporal
   .option('--regulars-only', 'Download only internals', caporal.BOOL, false)
   .option('--excludes <exclude>', 'Exclude a module', caporal.REPEATABLE, [])
   .option('--production', 'Leave only production files')
+  .option('--verbose, -v', 'Prints additional data on error', false)
```
Comment (alanggreen) [inline dev-scripts/get.js]: Can u explain to me what this does please

File: src/main/services.js [hunk 2]
```diff
@@ -79,10 +79,11 @@ exports.ServiceManager = class {
         if (!isRunning) {
           return Promise.try(init)
             .then(() => {
-              const child = execFile(options.executable, options.arguments || [], {
+              const child = spawn(options.executable, options.arguments || [], {
```
Comment (alanggreen) [inline src/main/services.js]: Seems like an excellent idea. A much better choice
https://www.freecodecamp.org/news/node-js-child-processes-everything-you-need-to-know-e69498fe970a/

File: dev-scripts/.get/get-caddy.js [hunk 3]
```diff
@@ -51,7 +51,9 @@ exports.getCaddy = function ({ directory, platform, arch }) {
           } else if (response.headers['content-type'] === 'application/x-gzip') {
             req
               .pipe(createGunzip())
-              .pipe(tar.extract(directory)
+              .pipe(tar.extract(directory, {
```
Comment (alanggreen) [inline dev-scripts/.get/get-caddy.js]: Does this change have to do with the shell

File: dev-scripts/get.js (hunk 1 above)
Comment (2roy999) [reply dev-scripts/get.js]: This is a script to bring all the modules, which runs when you use the command `yarn get` add an option `--verbose` so if a module's download fail one can look at a more informative error.

File: dev-scripts/.get/get-caddy.js (hunk 3 above)
Comment (2roy999) [reply dev-scripts/.get/get-caddy.js]: No it was suppose to go in two commits. (my bad)

**Pull request merged**